.github/workflows/ebpf.yml

```diff
@@ -31,57 +53,93 @@ jobs:
         echo "ref: ${{ github.event.client_payload.ref }}"
         echo "pr: ${{ github.event.client_payload.pr }}"
 
-  windows-perf:
+  build:
+    name: Build ebpf-for-windows
+    uses: microsoft/ebpf-for-windows/.github/workflows/reusable-build.yml@main
```

Comment:
This seems to be uploading 155 MB of build output that isn't actually used

Reply:
It's needed for decoding ETW traces with stack walking. At the very least we need the symbols for ebpfcore.sys. Agree we could probably have an option to add it to the MSI instead and skip uploading everything. I propose we update this in the ebpf-for-windows repo instead.